biome.json

```
@@ -0,0 +1,34 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "files": {
+    "ignoreUnknown": true,
+    "includes": ["**", "!**/.direnv", "!**/.next"]
+  },
+  "linter": {
+    "domains": {
+      "next": "recommended",
+      "react": "recommended"
+    },
+    "enabled": true,
+    "rules": {
+      "recommended": true,
+      "correctness": {
+        "noUnusedImports": "error"
+      },
+      "nursery": {
+        "useSortedClasses": {
+          "fix": "safe",
+          "level": "error",
+          "options": {
+            "functions": ["clsx", "cn", "cva", "tw", "tw.*"]
+          }
+        }
+      }
+    }
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": true,
+    "indentStyle": "space"
+  }
```

```suggestion
  "formatter": {
    "enabled": true,
    "formatWithErrors": true,
    "indentStyle": "space"
     "importSorting": true
     "quoteStyle": "double" 
  }
```
 import 순서와, quoteStyle 관련해서도 프리티어에서 설정하던 습관이 있어서, 여기에서도 추가하는걸 제안드려봅니다!!

---

- `importSorting`은 없는 옵션 같은데, 관련해서는 `assist/source/organizeImports`가 기본적으로 활성화 되어 있어서, import sort가 자동으로 되고 import sort가 되어 있지 않은 코드는 error로 표시해서 자동으로 fix도 가능해요 🤖 
- `quoteStyle`은 biome에서는 기본으로 double quote로 설정되어 있어서 따로 설정하지 않고 비워두었어요!